packages/react-dom/src/events/plugins/__tests__/ModernChangeEventPlugin-test.internal.js

```diff
@@ -53,7 +54,11 @@ describe('ChangeEventPlugin', () => {
         postMessageCallback(postMessageEvent);
       }
     };
-    jest.resetModules();
+    ReactFeatureFlags.enableModernEventSystem = true;
```

Comment:
I think... you don't need this test to be `.internal` anymore since the `enableModernEventSystem` flag is marked as *variant* in `ReactFeatureFlags.www-dynamic`.

I think you should be able to remove the `.internal` suffix on this (and the below test) and just use the new pragma comment:
```js
// @gate enableModernEventSystem
```

Reply:
I tried this locally just now, for some reason some tests in files still break with this change. Namely `EnterLeave`. To make sure I'm sane, I break the old legacy event system whilst running the change, so I can see if we are using the modern system or the legacy – and it seems we are still using the legacy with `// @gate enableModernEventSystem`

I'd rather tackle why that might be the case in another PR. :)